iosxr_test.py

```diff
@@ -0,0 +1,262 @@
+#!/usr/bin/python
+'''
+Author: Rich Wellum (richwellum@gmail.com)
+
+A tool to run some basic tests on a IOS XRv vagrant box, using pxssh module.
+
+This will verify both IOS XR Linux and IOS XR Console access.
+
+Is called from iosxr_iso2vbox post generation to verify sanity of
+the Virtualbox,
+
+Can also be run manually, like this:
+
+python iosxr_test.py iosxrv-fullk9-x64.box
+'''
+
+from __future__ import print_function
+import sys
+import pexpect
+from pexpect import pxssh
```

Comment:
pxssh isn't part of the python standard library right? You might want to add a note in the README stating that the user needs to install it.

(When this is reworked as a proper Python package, you can explicitly flag pxssh as a dependency.)